### ISSUE TITLE
Blur in Overview renderes above the desktop previews

### ISSUE THREAD

**Issue by Znert** [aunetx/blur-my-shell#283]
Hi, since a view days (weeks?) whenever I switch to the overview, the blurred background overlaps the desktop previews.

It seems to be consistent every time I boot my laptop.

Here is the broken view:

![Screenshot from 2022-06-06 10-44-42](https://user-images.githubusercontent.com/18900196/172128629-07e65cff-7df0-4f52-891b-6825799c2ae5.png)

And here it is fixed after I disable and re-enable the extension:

![Screenshot from 2022-06-06 10-45-41-obfuscated](https://user-images.githubusercontent.com/18900196/172128840-4e4ff09a-e552-4e95-9177-03fd125b5fcf.png)

It stays fixed until I reboot.

I'm on Manjaro with Gnome 42.1 using Wayland.

**Comment by pmoieni** [comment]
can confirm. this also happens on Fedora 36 with Gnome 42.2 on Wayland.

**Comment by Znert** [comment]
Interesting, I use Fedora 36 with Wayland with I believe the latest version of Gnome on my office laptop, but everything works there. Are you on Nvidia too? Which drivers are you using? I *think* I'm using the proprietary drivers on my office machine, but definitely the open source ones on my private Manjaro laptop.

**Comment by pmoieni** [comment]
> Interesting, I use Fedora 36 with Wayland with I believe the latest version of Gnome on my office laptop, but everything works there. Are you on Nvidia too? Which drivers are you using? I *think* I'm using the proprietary drivers on my office machine, but definitely the open source ones on my private Manjaro laptop.

Nope. I have an AMD 7400m GPU. The issue still present in version 39.

**Comment by Znert** [comment]
As an update, this issue now also happens on my Fedora work laptop and my Manjaro desktop. On my Manjaro laptop the issue still persists.

On my workstation I'm using Fedora 36 with Gnome 42.4.

**Comment by Znert** [comment]
Updated to Fedora 37/Gnome 43 on my workstation, issue still persists.

As a small updated, I can fix the blurred background by just *launching* the extensions app. Looks like something gets (re-)evaluated, which fixes the z-indexes?
